lib/generators/rodauth/templates/app/views/rodauth/tailwind/add_recovery_codes.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :title, rodauth.add_recovery_codes_page_title %>
```

Comment:
These have been removed in favour of the title instance variable that's now automatically set by rodauth-rails.

Reply:
fixed with: [101afbe](https://github.com/janko/rodauth-rails/pull/114/commits/101afbeac6751097d6aad074aafe3317b40afe4a)